filebeat/prospector/include/list.go

```diff
@@ -0,0 +1,8 @@
+package include
```

Comment:
I would suggest to move this directly under `filebeat` instead of `prospectr` as we ave it in other projects.